src/phase0/state_transition/mod.rs

```diff
@@ -842,9 +840,6 @@ pub fn get_beacon_proposer_index<
     const EPOCHS_PER_SLASHINGS_VECTOR: usize,
     const MAX_VALIDATORS_PER_COMMITTEE: usize,
     const PENDING_ATTESTATIONS_BOUND: usize,
-    const MAX_COMMITTEES_PER_SLOT: u64,
```

Comment:
so these were just extra generics? i suppose this will become clear as we use these functions elsewhere but something to keep an eye on!